### PR TITLE
SUPPORTESC-344 - Testing account requirements

### PR DESCRIPTION
**Pull request by redbeardaz**
Replace contact@zapier.com with integration-testing@zapier.com for
new test accounts.

Update email links to help center links.

https://zapierorg.atlassian.net/browse/SUPPORTESC-344

### CODE REVIEW

**Comment by redbeardaz** [comment]
Adding a new reviewer and tagging in @tylerrobertsonzapier to help me ship this 😁

**Review by tylerrobertsonzapier — APPROVED** [review]
These changes look good to me! I especially appreciate changing the `mailto:` links to refer to our contact form, that will help prevent a lot of incorrectly-routed tickets in the future.

File: docs/_legacy/docs.md [hunk 1]
```diff
@@ -2379,7 +2379,7 @@ Content-Type: application/pdf
 
 ### Advanced Streaming via Scripting
 
-At this time we do no support advanced streaming of files via the scripting platform (for example, uploading a file to receive an attachment ID that gets mixed into the normal JSON for POSTing). We may support that in the future, please send us an [email](mailto:contact@zapier.com) if you have questions!
+At this time we do no support advanced streaming of files via the scripting platform (for example, uploading a file to receive an attachment ID that gets mixed into the normal JSON for POSTing). We may support that in the future, please [let us know](https://developer.zapier.com/contact) if you have questions!
```
Comment (armchairlinguist) [inline docs/_legacy/docs.md:2382]:
```suggestion
At this time we do not support advanced streaming of files via the scripting platform (for example, uploading a file to receive an attachment ID that gets mixed into the normal JSON for POSTing). We may support that in the future, please [let us know](https://developer.zapier.com/contact) if you have questions!
```

File: docs/_partners/faq.md [hunk 2]
```diff
@@ -57,11 +57,11 @@ Can't find the answer you need here or in Zapier's [integration planning guide](
 
 Zapier's Platform team has a [community](https://community.zapier.com/developer-discussion-13) dedicated to helping developers with new and existing integrations. We're a worldwide team, so there's usually someone around to answer your questions. We'd love to hear about your experience building on Zapier.
 
-### Email Zapier's Platform Team
+### Contact Zapier's Platform Team
 
-For questions, email [partners@zapier.com](mailto:partners@zapier.com), where our Platform team can help resolve issues you encounter while building Zapier integrations, and our Partnerships team can help launch them.
+For questions about the developer platform and partnerships, [contact us through our help center](https://developer.zapier.com/contact), where our Platform Support team can help resolve issues you encounter while building Zapier integrations, and our Partnerships team can help any launch questions.
```
Comment (armchairlinguist) [inline docs/_partners/faq.md:62]:
```suggestion
For questions about the developer platform and partnerships, [contact us through our help center](https://developer.zapier.com/contact), where our Platform Support team can help resolve issues you encounter while building Zapier integrations, and our Partnerships team can help with any launch questions.
```

File: docs/_partners/integration-review-guidelines.md [hunk 3]
```diff
@@ -29,7 +29,7 @@ Before submitting your Zapier integration for review, make sure you are familiar
...
-* Provide an active demo account and login credentials, plus any other resources that might be needed to review your integration (e.g. login credentials to your product under [contact@zapier.com](mailto:contact@zapier.com) as the username/email and dummy credit card credentials used to test making purchases, if applicable to your integration). We test each trigger/action/search in your integration, so please provide us with access to the appropriate platforms and resources to do so.
+* Provide an active demo account and login credentials, plus any other resources that might be needed to review your integration (e.g. login credentials to your product under integration-testing@zapier.com as the username/email and dummy credit card credentials used to test making purchases, if applicable to your integration). We test each trigger/action/search in your integration, so please provide us with access to the appropriate platforms and resources to do so.
```
Comment (armchairlinguist) [inline docs/_partners/integration-review-guidelines.md:32]:
```suggestion
* Provide an active demo account and login credentials, plus any other resources that might be needed to review your integration (e.g. login credentials to your product under integration-testing@zapier.com as the username/email and dummy credit card credentials used to test making purchases, if applicable to your integration). We may need to test triggers, actions, or searches in your integration during review, so please provide us with access to the appropriate platforms and resources to do so.
```

**Review by armchairlinguist — APPROVED** [review]
Looks good, thanks for picking those up!

**Pull request merged**